### PR TITLE
Added logo in title 

### PR DESCRIPTION
**Pull request by saumyasrivastava08**
#124

### CODE REVIEW

**Comment by vaishali614** [comment]
@saumyasrivastava08 The logo is still not visible in title.

**Comment by saumyasrivastava08** [comment]
logo color is same as screen thats why its all black

**Comment by vaishali614** [comment]
> logo color is same as screen thats why its all black

No, there's no logo.

Capture

File: public/index.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
 
-    <link rel="icon" href="../assets/GCI.png" type="image/icon type" />
+    <link  href="%PUBLIC_URL%/GCI.png"  />
```
Comment (abdus) [inline public/index.html]: you have to add [`type` and `rel` to `<link>` tag](https://www.w3schools.com/tags/att_link_sizes.asp).
You can also rename `public/GCI.png` to `public/favicon.ico`, and the browser will take care of the rest.

**Review by abdus — CHANGES_REQUESTED** [review]

**Comment by saumyasrivastava08** [comment]
Screenshot (66)
 

visible in my

**Review by Manvityagi — APPROVED** [review]

**Pull request merged**